src/Symfony/Component/DependencyInjection/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 CHANGELOG
 =========
 
+6.1
+---
+ * Add method `getString` to `ParameterBag`
```

Comment:
missing blank line before, but more importantly, this file is not in the patched component

Reply:
Oh wow, I totally missed that. 🙈

Reply:
I fixed blank line, but understand, what the patched component?

Reply:
wow, I'm sorry, I fixed this :)